pkg/proxy/apis/config/v1alpha1/conversion.go

```diff
@@ -28,6 +28,10 @@ func Convert_config_KubeProxyConfiguration_To_v1alpha1_KubeProxyConfiguration(in
 		return err
 	}
 	out.WindowsRunAsService = in.Windows.RunAsService
+	out.Conntrack = v1alpha1.KubeProxyConntrackConfiguration(in.Linux.Conntrack)
+	out.OOMScoreAdj = in.Linux.OOMScoreAdj
+	out.NFTables.MasqueradeAll = in.Linux.MasqueradeAll
+	out.IPTables.MasqueradeAll = in.Linux.MasqueradeAll
```

Comment:
I'd probably make this conditional on `in.Mode` like with the other conversion func. (Ideally the mode-specific configs should all have default values when not using that mode...)